eZ/Publish/API/Repository/Tests/ContentServiceTest.php

```diff
@@ -1476,6 +1477,60 @@ public function testUpdateContentThrowsContentFieldValidationException()
         /* END: Use Case */
     }
 
+    /**
+     * Test for the updateContent() method.
+     *
+     * @see \eZ\Publish\API\Repository\ContentService::updateContent()
+     * @depends eZ\Publish\API\Repository\Tests\ContentServiceTest::testUpdateContent
+     */
+    public function testUpdateContentIgnoreIsRequiredOnNonUpdatedLanguages()
```

Comment:
I got lost when reading that method name :)
Maybe `testUpdateContentValidatorIgnoresRequiredFieldsOfNotUpdatedLanguages`